assets/css/style.css

```diff
@@ -260,3 +260,21 @@ a.btn-see-resume:active {
 }
 
 /*** Popup styling end **/
+.contact-form > span {
+  color: white;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  align-self: center;
+  padding-bottom: 5px;
+}
+
+.error {
+  margin-bottom: 15px;
+  height: 48px;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  border: 1px solid red;
+  padding: 16px;
+  color: #172b4d;
+}
```

Comment:
- [ ] [Optional] Just to make it more fun and context, make this error text a red one

Reply:
Thank you for your feed back